### ISSUE TITLE
Under 60fps recording despite being able to play and record 1080p60

### ISSUE THREAD

**Issue by Ryzym** [Xaymar/OBS-AMD-Advanced-Media-Framework#22]
So while recording Overwatch, I can play in 60fps and the shitty original AMD encoder can do 720p60 with no issues, this plugin can't do 1080p60 (highest I could get was 48fps). I've tried lowering the settings and setting the preset to Speed but it doesn't work. It also shows up choppy and low fps in the preview window. Should I attach a log to this when I get home later? I have to go out soon.

Edit: Forgot to add I tried with frameskipping turned on and off, no difference.

**Comment by Xaymar** [comment]
That is strange, I have a weaker CPU and barely hit 12% usage on 1080p60. What kind of RAM do you have? DDR3/4, 1333Mhz/1600Mhz/1866Mhz/...?

Edit: The encoder plugin uses assisted paralellization, so if your memory sucks that affects usage.

**Comment by Xaymar** [comment]
Additionally to the above, make sure that you are using NV12. All others are non-native and need to be converted by AMF first.

**Comment by Xaymar** [comment]
You might be encountering the same issue as in #18

**Comment by Ryzym** [comment]
I have DDR3 RAM, and issue #18 might be the issue because I did upgrade my drivers about 2-3 days ago but never upgraded the plugin until last night.

**Comment by Xaymar** [comment]
Yeah, 16.7.3 has massively increased submission time. I'm not sure what it does, but it feels exactly the same as media foundation now. Ultra high cpu usage with no real benefit.

**Comment by Ryzym** [comment]
Well, I'll just continue with Plays.tv/Overwolf until you figure something out. Thanks for all the great work btw. Really appreciate this plugin.

**Comment by Xaymar** [comment]
There's nothing I can do here, as the issue is with the Media Framework SDK. Until that is updated, there is nothing i can "figure out".

**Comment by Xaymar** [comment]
What you can try though is to install 16.7.3/16.7.2 on a clean slate - wipe drivers with DDU in safe mode and then install again. That reduced the submission time by ~12ms for me.

**Comment by Ryzym** [comment]
Used DDU and reinstalled 16.7.2 and still getting massive CPU usage when with no games running or being recorded. Just a black screen hits 30%+.

**Comment by Xaymar** [comment]
I did the same for 16.7.3 and it went away for me. If you did upgrade to 16.7.3, you can't go back to 16.7.2 unfortunately - whatever AMD sends to the VCE chip sticks around.

**Comment by Ryzym** [comment]
So do I have to clean install 16.7.3 or am I stuck waiting until they update the SDK? I don't mind waiting as long as I know that what I have is not going to work.

**Comment by Xaymar** [comment]
That is so far the only working solution that I found on all my PCs. No other driver returned the submission time back to normal.

**Comment by Ryzym** [comment]
I installed 16.7.3 and it dropped down between 25-30% usage but it's not going any farther down. I have no idea what causing it. Would a small log file help?

Edit: I was recording with OpenCL and RGB.. I changed them to Direct3D 11 and NV12 and my CPU usage went down to 7% with no game being recorded. Sorry, I'm an idiot..

Edit 2: Now that I can use the plugin without frying my CPU, it can't reach 60fps anymore.. Just recorded a bit of Overwatch to test and the highest it reached was 48fps, despite the game playing in 60.
